models/comments-models.js

```diff
@@ -28,3 +28,18 @@ exports.insertComment = (newComment, review_id) => {
       return result.rows[0];
     });
 };
+
+exports.fetchCommentById = (comment_id) => {
```

Comment:
Everything great here except the name of this function, which is misleading. It is deleting rather than fetching. I would suggest removeCommentById()